src/rating-indicator.scss

```diff
@@ -0,0 +1,119 @@
+@import "./new-settings";
+@import "./mixins";
+@import "./icon";
+
+$block: #{$fd-namespace}-rating-indicator;
+
+.#{$block} {
+  $ratedColor: var(--sapContent_RatedColor);
+  $unratedColor: var(--sapContent_UnratedColor);
+  $active-star-hover-opacity-transaction: opacity $fd-animation-speed ease-in;
```

Comment:
Please use the naming convention of the project

Reply:
fixed